Libraries_API.ipynb

```diff
@@ -713,6 +664,151 @@
     "    -X DELETE"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### /query/&lt;library_id&gt;\n",
+    "Add or remove documents the library specified by `library_id` (string) is the unique identifier of the library, as specified in the `/libraries` GET response.\n",
```

Comment:
The first half (before the comma) of this line needs some editing - I think two clauses got smushed together weirdly